Deepanshu_Balani/project2 tictactoe/assets/tictac.css

```diff
@@ -0,0 +1,41 @@
+td{
+border:2px solid brown;
+height:100px;
+width:100px;
+text-align: center;
+vertical-align: middle;
+font-family: Georgia, 'Times New Roman', Times, serif;
+font-size: 70px;
+cursor:pointer;
+}
+.container {
+    display: flex;
+    background:blue;
+    align-content: center;
+    justify-content: center;
+    flex-direction: column;
```

Comment:
use short hand property to have clean code
instead of flex-direction, always use flex-flow

Reply:
flex-flow used instead of flex-direction